docs/DeveloperGuide.md

```diff
@@ -498,6 +498,70 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding a new student
+
+**Note:** You are encouraged to follow the test cases in order to see the expected results
+
+1. Adding a new student with all parameters specified
+
+   1. Simply type `add` into the command box and click `Enter`.
+
+   1. Copy the example shown together with the error message in the result display area.
+
+   1. Test case: `add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/Sunday-1000-1200 
+   s/Biology r/120 paid/6000 owed/3000`.
+
+1. Adding a new student with only compulsory parameters specified, order changed and case-insensitive command words
```

Comment:
I have a suggestion:

"Adding a new student with these conditions: 1) only compulsory parameters are specified 2) order of parameters are changed 3) using both lowercase and uppercase letters for command word"